src/ray/observability/ray_event_recorder.cc

```diff
@@ -86,5 +93,7 @@ void RayEventRecorder::AddEvents(
   }
 }
 
+bool RayEventRecorder::isEnabled() { return RayConfig::instance().enable_ray_event(); }
```

Comment:
looks like the method is unnecessary -- it's only used internally and it's a one liner, so I'd just inline the config check